pkgs/development/ocaml-modules/promise_jsoo/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, fetchurl, buildDunePackage, ocaml, js_of_ocaml, ppxlib, js_of_ocaml-ppx, gen_js_api }:
```

Comment:
```suggestion
{ lib, fetchurl, buildDunePackage, js_of_ocaml, ppxlib, js_of_ocaml-ppx, gen_js_api }:
```

Reply:
Can you help me understand this suggestion?

Reply:
The derivation does not use the `ocaml` parameter. So there is no need to have it as a parameter.

Reply:
Oh crap. I misread the diff as you adding the `ocaml` parameter.

Reply:
It seems that `ocaml` is back into the parameter set.

Reply:
Done.